ACHP/PHEHX.py

```diff
@@ -419,12 +419,12 @@ def collect(cellList,tag,tagvalue,out):
             else:
                 self.DT_sc_h=self.Tbubble_h-self.Tout_h
         
-    def eNTU_CounterFlow(self,Cr,Ntu):
-        """
-        This function returns the effectiveness for counter flow fluids
-        """
-        return ((1 - exp(-Ntu * (1 - Cr))) / 
-            (1 - Cr * exp(-Ntu * (1 - Cr))))
+#     def eNTU_CounterFlow(self,Cr,Ntu):

```

Comment:
This function is not used in the code. We use epsilon from Q/Qmax to calculate NTU. See
https://github.com/TSTK/ACHP/blob/master/ACHP/PHEHX.py#L465
and
https://github.com/TSTK/ACHP/blob/master/ACHP/PHEHX.py#L602
I commented it out just in case we might need it later